dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatCompletionExtensions.cs

```diff
@@ -44,5 +44,20 @@ public static class ChatCompletionExtensions
         string input,
         PromptExecutionSettings? executionSettings = null,
         CancellationToken cancellationToken = default)
-            => chatCompletion.GetStreamingContentAsync<StreamingContent>(input, executionSettings, cancellationToken);
+            => chatCompletion.GetStreamingContentAsync<StreamingContent>(chatCompletion.CreateNewChat(input), executionSettings, cancellationToken);
+
+    /// <summary>
+    /// Get asynchronous stream of <see cref="StreamingContent"/>.
+    /// </summary>
+    /// <param name="chatCompletion">Chat completion target</param>
+    /// <param name="input">The input string that will be used as the instructions of the chat</param>
+    /// <param name="executionSettings">Request settings for the completion API</param>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
+    /// <returns>Streaming list of different completion streaming result updates generated by the remote model</returns>
+    public static IAsyncEnumerable<T> GetStreamingContentAsync<T>(
+        this IChatCompletion chatCompletion,
+        string input,
```

Comment:
What do we want the behavior to be if you pass a null input? If that's desirable to allow, this should be `string?`. If that should be discouraged, we should have `Verify.NotNull(string);` at the beginning of the implementation.

Reply:
Agree, LLM should have always a valid prompt which means it should not be null or whitespace.